popup.js

```diff
@@ -1,10 +1,83 @@
-const body = document.querySelector('body');
-// const overlay = document.querySelector('.overlay');
 const mainBg = document.querySelector('.main-bg');
-const buttons = document.querySelectorAll('.btn');
-
 const modalWrapper = document.querySelector('.modal-wrapper');
 
+const worksContainer = document.querySelector('#work-section');
+
+// define cards elements to be created unto page
+const cardObjects = [
+  {
+    img: './images/Snapshoot Portfolio.png" alt="tonic image',
+    title: 'Tonic',
+    type: 'CANOPY',
+    dev: 'Back End Dev',
+    year: 2015,
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    tech: ['HTML', 'CSS', 'JavaScript'],
+    btn: 'See projects',
+  },
+
+  {
+    img: './images/Snapshoot Portfolio (1).png',
+    title: 'Multi-Post Stories',
+    type: 'CANOPY',
+    dev: 'Back End Dev',
+    year: 2015,
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    tech: ['HTML', 'CSS', 'JavaScript'],
+    btn: 'See projects',
+  },
+  {
+    img: './images/Snapshoot Portfolio (2).png',
+    title: 'Tonic',
+    type: 'CANOPY',
+    dev: 'Back End Dev',
+    year: 2015,
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    tech: ['HTML', 'CSS', 'JavaScript'],
+    btn: 'See projects',
+  },
+
+  {
+    img: './images/Snapshoot Portfolio (3).png',
+    title: 'Multi-Post Stories',
+    type: 'CANOPY',
+    dev: 'Back End Dev',
+    year: 2015,
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    tech: ['HTML', 'CSS', 'JavaScript'],
+    btn: 'See projects',
+  },
+];
+// create card unto page
+function createCard() {
+  /* eslint-disable no-plusplus */
+  for (let card = 0; card < cardObjects.length; card++) {
+    worksContainer.innerHTML += ` 
+    <div class="card two">
+    <div class="card-img tonic">
+        <img src="${cardObjects[card].img}" alt="tonic image">
+    </div>
+    <div class="card-description">
+        <h2 class="card-title">${cardObjects[card].title}</h2>
+        <ul class="canopy">
+            <li id="canopy-text">${cardObjects[card].type}</li>
+            <li>${cardObjects[card].dev}</li>
+            <li>${cardObjects[card].year}</li>
+        </ul>
+        <p class="card-details">
+        ${cardObjects[card].description}</p>
+        <ul class="card-tools">
+            <li>${cardObjects[card].tech[0]}</li>
+            <li>${cardObjects[card].tech[1]}</li>
+            <li>${cardObjects[card].tech[2]}</li>
+        </ul>
+        <button type="submit" class="btn project-btn btn-${card + 1}">${cardObjects[card].btn}</button>
+    </div>
+</div>
+    `;
+  }
+}
```

Comment:
Weldone on getting this far, kindly note that while applying the changes introduced by the previous reviewer, the `project section` of your portfolio is not populated on the screen, Please while implementing the change, endeavour to check your final outlook to be sure it displays

This is the current behaviour of your project :

https://github.com/Shabiblac/Microverse-Portfolio-Design/assets/20537712/9193394f-069d-4570-a481-3300ff1efa6a